chpl/chpl-service/src/main/java/gov/healthit/chpl/manager/ProductManager.java

```diff
@@ -106,6 +110,9 @@ public boolean exists(Long id) {
     @Transactional(readOnly = false)
     @PreAuthorize("@permissions.hasAccess(T(gov.healthit.chpl.permissions.Permissions).PRODUCT, "
             + "T(gov.healthit.chpl.permissions.domains.ProductDomainPermissions).CREATE)")
+    @CacheEvict(value = {
+            CacheNames.COLLECTIONS_LISTINGS
```

Comment:
Should probably evict the Product Names cache: `CacheNames.PRODUCT_NAMES`

Reply:
added